manifests/07-operator.yaml

```diff
@@ -42,6 +42,10 @@ spec:
         env:
         - name: IMAGE
           value: docker.io/openshift/origin-console:latest
+        - name: OPERATOR_IMAGE
+          value: docker.io/openshift/origin-console-operator:v4.0
+        - name: OPERATOR_IMAGE_VERSION
```

Comment:
Did we know if we also need `OPERAND_IMAGE_VERSION`? I assume not, just asking.

Reply:
They are not using this envvar anywhere else, so I assume we dont 👍

Reply:
neat :)